src/benchmarks/eina/eina_bench.c

```diff
@@ -111,7 +111,7 @@ main(int argc, char **argv)
       case 2:
          for (i = 0; etc[i].bench_case; ++i)
            {
-              if (strcasecmp(etc[i].bench_case, argv[1]))
+              if (evil_strcasecmp(etc[i].bench_case, argv[1]))
```

Comment:
You do not use `evil_*` explicitly. The idea of Evil is that noone knows about Evil despiste including it as a header.